typings/capitano/index.d.ts

```diff
@@ -41,6 +41,7 @@ declare module 'capitano' {
 		permission?: 'user';
 		root?: boolean;
 		primary?: boolean;
+		hidden?: booleanl;
```

Comment:
`booleanl` - ending with `l` - is that a thing? :-)

Reply:
Huh.  That's not checked by the compiler? TIL.

Reply:
Something went wrong, because it is meant to be type checked:
https://github.com/balena-io/balena-cli/blob/v11.30.13/tsconfig.dev.json#L7

Also, if all manual and automated tests passed despite the typo, is the line even necessary?

Reply:
@pdcastro It's not technically necessary anymore (and I guess that's why it wasn't checked, though I'm suprised even unused definitions don't undergo some kind of checking).  I added it, because I had initial converted the capitano command to ts, before the oclif conversion (I must have added the typo sometime after that I suppose, as it was working). 

I decided to leave it in, as technically the definition is not complete without it.